DependencyInjection/Configuration.php

```diff
@@ -216,7 +216,7 @@ private function addFilesystemSection(ArrayNodeDefinition $node)
                                 ->end()
                                 ->scalarNode('region')->defaultValue('s3.amazonaws.com')->end()
                                 ->scalarNode('version')->defaultValue('latest')->end()
-                                ->scalarNode('aws_sdk_version')->defaultValue(2)->end()
+                                ->scalarNode('sdk_version')->defaultValue(2)->end()
```

Comment:
Same here. We don't need to see that, fix it up into the commit that introduced this.